.travis.yml

```diff
@@ -2,20 +2,25 @@ language: php
 matrix:
   include:
     - php: 7.0
+      env: ES=1
+    - php: 7.0
+      env: ES=2
+    - php: 5.6
+      env: ES=1
     - php: 5.6
+      env: ES=2
 sudo: false
 before_install:
   - export NEOS_TARGET_VERSION=2.3
   - cd ..
-  - wget --no-check-certificate https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.5.zip && unzip elasticsearch-1.7.5.zip
-  - mv elasticsearch-1.7.5 elasticsearch
+  - if [ "$ES" = 1 ]; then wget --no-check-certificate https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-1.7.5.zip && unzip elasticsearch-1.7.5.zip && mv elasticsearch-1.7.5 elasticsearch; fi
+  - if [ "$ES" = 2 ]; then wget --no-check-certificate https://download.elastic.co/elasticsearch/elasticsearch/elasticsearch-2.4.3.zip && unzip elasticsearch-2.4.3.zip && mv elasticsearch-2.4.3 elasticsearch; fi
   - cd elasticsearch
   - bin/elasticsearch -d
   - cd ..
   - git clone https://github.com/neos/neos-base-distribution.git -b ${NEOS_TARGET_VERSION}
   - cd neos-base-distribution
-  - composer require flowpack/elasticsearch-contentrepositoryadaptor
-  - composer require typo3/typo3cr-search
+  - composer require typo3/typo3cr-search:2.1.2 flowpack/elasticsearch-contentrepositoryadaptor
```

Comment:
Why that specific version of `typo3/typo3cr-search`? And why not `neos/content-repository-search`?